settings.gradle

```diff
@@ -11,7 +11,6 @@ include ':app',
         ':libtrip-notification',
         ':liblogger',
         ':libnavigation-util',
-        ':libdirections-session',
```

Comment:
Why don't we just remove `:libdirections-session` here, and keep developing this module locally?  Doesn't it work?

Reply:
If it would live in a separate module, we couldn't make the implementation `internal`.

Reply:
It can be `public` but under an `internal` package.

Reply:
Also, we need to remove this https://github.com/mapbox/mapbox-navigation-android/blob/f05958cb6aed1c349497cc9ee89bf73ee0fa69d2/Makefile#L18 as module was removed.

Reply:
I think we can keep it simpler by just moving the code.

Reply:
As for removing the line, covered in https://github.com/mapbox/mapbox-navigation-android/pull/2325.

Reply:
> I think we can keep it simpler by just moving the code.

Sounds good. I was asking because at some point we'll need to have the module, right? And as we already had it set up, thought that could be faster to expose it when needed ¯\\\_(ツ)\_/¯